src/tools/compiletest/src/runtest.rs

```diff
@@ -375,9 +375,10 @@ impl<'test> TestCx<'test> {
         // if a test does not crash, consider it an error
         if proc_res.status.success() || matches!(proc_res.status.code(), Some(1 | 0)) {
             self.fatal(&format!(
-                "test no longer crashes/triggers ICE! Please give it a mearningful name, \
+                "Crashtest no longer crashes/triggers ICE, horray! Please give it a meaningful name, \
             add a doc-comment to the start of the test explaining why it exists and \
-            move it to tests/ui or wherever you see fit."
+            move it to tests/ui or wherever you see fit. Adding 'Fixes #<issueNr>' to your PR body \
+            and commit message ensures that the correspondig ticket is auto-closed upon merge."
```

Comment:
commit messages should probably be avoided to avoid spamming the issues with duplicates from rebases and forks

```suggestion
            and commit message ensures that the corresponding ticket is auto-closed upon merge."
```